testsuite/tests/runtime-errors/Makefile

```diff
@@ -30,8 +30,9 @@ compile:
 	    $(OCAMLOPT) -w a -o $$F.native$(EXE) $$f; \
 	  fi; \
 	done
+	$(if $(findstring win32,$(UNIX_OR_WIN32)),:, \
```

Comment:
This doesn't look right - surely config/s-nt.h needs updating to define HAS_STACK_OVERFLOW_DETECTION for the Win64?